main.js

```diff
@@ -0,0 +1,25 @@
+const menuButton = document.querySelector('.mobile_menu');
+const mobileMenu = document.querySelector('#menu_top');
+const closeButton = document.querySelector('#close-button');
+
+function openMobileMenu() {
+  mobileMenu.style.display = 'block';
+}
+function closeMobileMenu() {
+  mobileMenu.style.display = 'none';
+}
+mobileMenu.style.display = 'none';
+menuButton.addEventListener('click', openMobileMenu);
+closeButton.addEventListener('click', closeMobileMenu);
+
+const menuItems = document.querySelectorAll('#menu_top a');
+
+menuItems.forEach((item) => {
+  item.addEventListener('click', function () {
+    closeMobileMenu();
+    const target = this.getAttribute('href');
+    document.querySelector(target).scrollIntoView({
+      behavior: 'smooth',
```

Comment:
- [OPTINAL] Just as I suggested before, please try to implement this in the CSS file, the current code is giving an error in the console which is not code for user experience. :smile: 

- ERROR SCREENSHOT:
![Screenshot from 2023-04-21 23-13-08](https://user-images.githubusercontent.com/107632780/233742537-8ea21087-9da9-4ac9-9a4e-bf99359d0ba6.png)